### PR TITLE
Update logo and colors

### PR DESCRIPTION
**Pull request by maanex**
typo in commit message 💀

### CODE REVIEW

**Review by stormofice — APPROVED** [review]
Looks good!
I have some concerns regarding accessibility on the (currently disabled) "More Details" button, but these should be resolved as soon as we have that feature implemented.

**Review by StealWonders — APPROVED** [review]

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ const render = function () {
         <link rel="manifest" href="/manifest.json" />
         <link rel="apple-touch-icon" href="/icon.png"></link>
         <meta name="theme-color" content="#fff" />
+        <link rel="shortcut icon" href="/favicon-32.png" />
```
Comment (GamerBene19) [inline src/pages/_document.tsx]: > Warning: The shortcut link type is often seen before icon, but this link type is non-conforming, ignored and web authors must not use it anymore. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

File: src/pages/_document.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ const render = function () {
         <link rel="manifest" href="/manifest.json" />
         <link rel="apple-touch-icon" href="/icon.png"></link>
```
Comment (GamerBene19) [inline src/pages/_document.tsx]:
```suggestion
        <link rel="apple-touch-icon" sizes="180x180" href="/favicon-180.png" />
```

File: src/pages/_document.tsx (hunk 1 above)
Comment (GamerBene19) [reply src/pages/_document.tsx]: It would be nice if you could move this line up so that all (two) lines containing icons are next to each other

**Review by GamerBene19 — CHANGES_REQUESTED** [review]
Great contribution! We finally get a proper icon :partying_face: :tada: 
However I have noticed a few things that should be changed (see comments).
Additionally I'm really not a fan of the white text, but as previously mentioned that might be an issue for the future.

Also it would be great if you could amend your commit to fix the typo in the message since because we're rebasing to merge it will end up on main 1:1.

**Review by GamerBene19 — APPROVED** [review]

**Pull request merged**